packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -0,0 +1,1116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as ts from "typescript";
+import XRegExp = require("xregexp");
```

Comment:
Is it possible to import this using `import XRegExp from xregexp`

Reply:
I suspect it needs to use this type of import.

Reply:
I'm not sure that's the case?

Generally, it's probably best to use `import` unless there is a really good reason to use require. 

https://stackoverflow.com/questions/46677752/the-difference-between-requirex-and-import-x

Reply:
This is not just a require() statement. It is a TypeScript import statement which is compatible with require. See https://www.typescriptlang.org/docs/handbook/modules/reference.html#export--and-import--require